ts/widgets-base.d.ts

```diff
@@ -1389,7 +1389,12 @@ declare module DevExpress.ui {
         /** @docid_ignore dxLoadPanelOptions_animation_hide */
```

Comment:
Rid of `@docid_ignore` comments contains `animation_hide` and `animation_show` options